content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,115 @@
+title: All about my Outreachy internship with Mixxx DJ: Intro
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2020-12-01 23:15:00
+
+#### A little bit about me.
```

Comment:
This should be below the introduction paragraph. Also, please use headline level `##` instead. These levels should be semantic. If you feel these are too big we can customize it in the CSS.

Also, headlines/images in the excerpt on overview pages look weird, so please add a `summary: ...` manually to the metadata in that case.

Reply:
Is there a way I can have the bullet and the point on the same line?  

![image](https://user-images.githubusercontent.com/28389950/103343211-a051c880-4a9c-11eb-9ab8-5be70bf01094.png)

Reply:
Yes: https://mixxx.org/news/2020-07-15-new-in-2-3-cmake/
Check the source code of the article in the GitHub repo :D

Reply:
Thank you. It has worked.